playbooks/tests/tasks/integration.yml

```diff
@@ -105,6 +105,43 @@
     volume: turtle-vol
   when: cinder.enabled|bool
 
+- name: instance is active
+  shell: . /root/stackrc; openstack server show {{ turtle_stack.openstack.id }} | grep ACTIVE
+
+- name: instance is running
+  shell: . /root/stackrc; openstack server show {{ turtle_stack.openstack.id }} | grep Running
+
+- name: instance ready for login
```

Comment:
probably we should have retry logic, 6 times with 10sec delay between each retry.